src/components/Inputs/inputs.scss

```diff
@@ -1,28 +1,51 @@
-.user-inputs {
+.submit-button-div {
     display: flex;
-    flex-direction: column;
-    gap: 20px;
+    justify-content: center;
+    
+    & .submit-button {
+        color: #fff;
+        border: 2px solid #fff;
+        border-radius: 10px;
+        padding: .5rem 1rem;
+        background-color: var(--buttonColor);
+        text-align: center;
+    }
+    
+    & .submit-button:hover {
+        background-color: var(--buttonHover);
+        cursor: pointer;
+    }
 }
 
-.weekday-input, .precip-input, .wind-input {
+.work-form {
     display: flex;
     flex-direction: row;
-    justify-content: space-between;
-}
+    justify-content: space-evenly;
+    padding-bottom: 1rem;
 
-.submit-button-div {
-    display: flex;
-    justify-content: center;
-}
 
-.submit-button {
-    background-color: #677680;
-    border: none;
-    padding: 1rem;
-}
+    & .work-form-labels {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        gap: 8px;
+    }
 
-.submit-button:hover {
-    background-color: #677680;
-    color: #fff;
-    cursor: pointer;
-}
+    & .work-form-inputs {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+    }
+
+    & .work-form-input-button {
```

Comment:
I noticed there's some inconsistencies between the buttons.  If you want all your buttons to be a certain color etc it would be better to make a Button component.  That way there'd be no repeating CSS and if you want to change the style of your whole website it's as simple as changing the Button component.